app/src/main/java/com/dsvag/currencyexchanger/data/adapters/CoinAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package com.dsvag.currencyexchanger.data.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.dsvag.currencyexchanger.R
+import com.dsvag.currencyexchanger.data.models.latest.Coin
+import kotlinx.android.synthetic.main.row_coin.view.*
+
+
+class CoinAdapter() : RecyclerView.Adapter<CoinAdapter.CoinViewHolder>() {
+
+    private var data: MutableList<Coin> = ArrayList()
+
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CoinViewHolder {
+        return CoinViewHolder(
+            LayoutInflater.from(parent.context).inflate(R.layout.row_coin, parent, false)
+        )
```

Comment:
А почему не через `ViewBinding`?
```suggestion
        val inflater = LayoutInflater.from(parent.context)
        return CoinViewHolder(
            RowCoinBinding.inflate(inflater, parent, false)
        )
```